src/Components/Layout/layout.module.css

```diff
@@ -25,10 +23,16 @@ aside {
 .switchContainer {
   display: flex;
   justify-content: center;
-  width: 85%;
+  width: 90%;
 }
 
 .asideContainer {
   height: auto;
-  width: 15%;
+  width: 5%;
+  transition: all 0.3s;
 }
+
+.asideContainer:hover{
+  width:15%;
+  transition: all 0.3s;
+}
```

Comment:
i think that you should use translate to modify the navbar, suggestion:
.asideContainer {
  height: auto;
  width:15%;
  transform: translateX(-8rem);
  transition: all 0.3s;
}

.asideContainer:hover{
  transform: translateX(0);
  transition: all 0.3s;
}
this way is the same effect but just you would have modify again icons of navbar